eveonline/managers.py

```diff
@@ -228,6 +228,10 @@ def get_character_by_id(char_id):
         return None
 
     @staticmethod
+    def get_characters_by_api_id(api_id):
```

Comment:
Meh, it works.

I've been on a crusade to delete these methods that do nothing. I will resume one day and nuke this one too.